src/alchemlyb/parsing/parquet.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+
+from . import _init_attrs
+
+
+@_init_attrs
+def extract_u_nk(path, T):
+    r"""Return reduced potentials `u_nk` from a pandas parquet file.
+
+    The parquet file should be serialised from the dataframe output
+    from any parser with command
+    (`u_nk_df.to_parquet(path=path, index=True)`).
+
+    Parameters
+    ----------
+    path : str
+        Path to parquet file to extract dataframe from.
+    T : float
+        Temperature in Kelvin the simulations sampled.
+
+    Returns
+    -------
+    u_nk : DataFrame
+        Potential energy for each alchemical state (k) for each frame (n).
+
+
+    Note
+    ----
+    pyarraw serializers would handle the float or string column name fine but will
+    convert multi-lambda column name from `(0.0, 0.0)` to `"('0.0', '0.0')"`.
+    This parser will restore the correct column name.
+
+
+    .. versionadded:: 2.1.0
+
+    """
+    u_nk = extract_dHdl(path, T)
```

Comment:
confusing, just be explicit
```suggestion
    u_nk = pd.read_parquet(path)
```